ci/src/cI_web_templates.ml

```diff
@@ -51,9 +51,9 @@ let gh_target_url = function
   | `Ref r ->
     let { CI_projectID.user; project } = CI_github_hooks.Ref.project r in
     let id = CI_github_hooks.Ref.name r in
-    match Datakit_path.unwrap id with
-    | "tags" :: id -> Fmt.strf "https://github.com/%s/%s/releases/tag/%s" user project (String.concat ~sep:"/" id)
-    | _            -> Fmt.strf "https://github.com/%s/%s/tree/%a" user project Datakit_path.pp id
+    match String.cut ~sep:"tags/" id with
```

Comment:
This is wrong. `tags` has to be the *first* component. Using strings is inviting these kinds of mistakes.